### DISCUSSION TITLE
Error after executing bin/console kimai:bundle:kiosk:install while installing Plugin on a Synology NAS with DSM7

### DISCUSSION THREAD

**Discussion by niklasfandrich** [kimai/kimai#5032] [Docker]
Hello all,

I want to install the [ImportBundle](https://github.com/kevinpapst/ImportBundle) on my Synology DS224+ running DSM7.

According to the documentation, the last step is to run the command ```bin/console kimai:bundle:kiosk:install```. I understand that it's an example and need to be modified (according to the plugin I'm about to install). But I'm a bit confused about what _exactly_ is required at this point. There doesn't seem to be a kimai command like ```kimai:bundle``` when I do ```bin/console kimai``` (to get a list of existing commands). I receive the following error when I execute the command:

> There are no commands defined in the "kimai:bundle:kiosk" namespace.

Does anyone have an idea what I need to do?

Thanks in advance!
Niklas

Used documentation: https://www.kimai.org/documentation/synology.html#installing-plugins

**Comment by kevinpapst** [comment]
See https://www.kimai.org/store/keleo-importer-bundle.html

Click on the installation tab
![image](https://github.com/user-attachments/assets/c6e1ab71-84d4-4de3-bb84-a24a96952ada)

If it does not mention any command, it is not necessary.

**Reply by niklasfandrich** [reply]
Thank you, Kevin. I did a clean installation again and I'm still not able to see the Plugin, also when I do a cache reload as described on the Plugin page. Might be a problem regarding file access. I'll dig a little deeper how it works in an Docker environment and maybe post an update later.

**Reply by kevinpapst** [reply]
The ImportBundle has NO installation command. Does the folder name equals /opt/kimai/var/plugins/ImportBundle/ ?

**Comment by BeckeBauer** [comment]
Open container manager (docker) app under DSM7 -> choose the `container` tab -> select your kimai container -> open the `Action` drop-down menu and select `open terminal` -> click on `create` -> select bash -> change to /opt/kimai -> execute `php bin/console kimai:bundle:kiosk:install`

![image](https://github.com/user-attachments/assets/b1db06c6-0746-4fcf-9cd1-df8e35d22199)

If nothing happens, check if your plugins are present under /opt/kimai/var/plugins

**Comment by alex0005** [comment]
Hi niklasfandrich, could you please share a readme how to install kimai on docker?

Until now I didn't have success.

according:
https://www.kimai.org/documentation/synology.html

"Next steps are not documented, if you try it, please share what you did, so we can adapt the docs! "

That would be very kindly!

Alex
